ci-operator/step-registry/interop-tests/cnv-tests-e2e-deploy/interop-tests-cnv-tests-e2e-deploy-commands.sh

```diff
@@ -36,7 +36,28 @@ cd /tmp/cnv-ci || exit 1
 
 # Overwrite the default configuration file used for testing
 export KUBEVIRT_TESTING_CONFIGURATION_FILE='kubevirt-tier1-ocs.json'
-make deploy_test || exit_code=$?
+
+# The default storage class used in the testing configuration file is 'ocs-storagecluster-ceph-rbd', at
+# https://github.com/openshift-cnv/cnv-ci/blob/master/manifests/testing/kubevirt-tier1-ocs.json, that is kept for
+# compatiblity with the current test configs at the time of writing this snippet.
+# The KUBEVIRT_STORAGECLASS_NAME is set to 'ocs-storagecluster-ceph-rbd' in the 'cnv-tests-e2e-deploy' step by default too.
+# Some test configurations like KubeVirt testing on ARM64 cannot use the storage class 'ocs-storagecluster-ceph-rbd' as it
+# is not available on the ARM64 nodes. Users can now set the storage class name to be used in the prow test config definition,
+# avoiding the need to modify these values in different repositories.
+cat > manifests/testing/kubevirt-tier1-ocs.json <<EOF
```

Comment:
Don't patch the file in the CI, create the file in the source repo, and overwrite `KUBEVIRT_TESTING_CONFIGURATION_FILE` with 
```
export KUBEVIRT_TESTING_CONFIGURATION_FILE=${KUBEVIRT_TESTING_CONFIGURATION_FILE:-kubevirt-tier1-ocs.json}
```

Reply:
I would avoid this as I think this file shouldn't be provided by another repository at all IMHO and in the future, we may deprecate it in favor of a "prow-driven" configuration.

Reply:
Until then. For now, keep it as we do it. We do the same with T2 test. I would patch the source code only if I don't have control over the source repo. But in our case, we do.

Reply:
Sure I'll keep that file for now.

I already considered and discarded the option you are proposing because I think exposing an environment variable in a step just to change the path of a file we can easily touch is not a good choice: to the eye of the user (test config) it's a lower-level and unclear information that means nothing but an internal test logic change of behavior the devs of the step only should be aware of. 

Instead, the storage class to use looks like the proper information to dynamically fill in this repo's steps.

Reply:
We test with multi-storage providers, and I don't want the config to be in the caller script.

Reply:
Can you clarify a proper reason not to do this?

Reply:
This script is just a simple launcher, it should not contain run-time specific config, the caller (the one call the test) should set the flags. I want the launcher to be generic.